### PR TITLE
DOC: fix some numpydoc / Sphinx issues.

### PR DESCRIPTION
**Pull request by rgommers**
Main one is to remove the duplicate Notes section, that doesn't work
with numpydoc master at the moment.

### CODE REVIEW

File: doc/source/tutorial/ndimage.rst [hunk 1]
```diff
@@ -461,7 +461,7 @@ information).
   object). The :func:`generic_filter1d` function iterates over the
   lines of an array and calls :func:`function` at each line. The
   arguments that are passed to :func:`function` are one-dimensional
-  arrays of the :ctype:`tFloat64` type. The first contains the values
+  arrays of the :c:type:`tFloat64` type. The first contains the values
```
Comment (ev-br) [inline doc/source/tutorial/ndimage.rst]: OT: What's `tFloat64`, is it a somehow known notation?

**Comment by ev-br** [comment]
This is an improvement regardless of a quibble about `tFloat64`. Merging, thanks Ralf.

File: doc/source/tutorial/ndimage.rst (hunk 1 above)
Comment (person142) [inline doc/source/tutorial/ndimage.rst]: I think it's an old `numarray` type:

https://heim.ifi.uio.no/~inf3330/scripting/doc/python/NumPy/numarray/node48.html

Comment (rgommers) [inline doc/source/tutorial/ndimage.rst]: Here's the left-over numarray remnant that needs to be cleaned up: https://github.com/scipy/scipy/blob/master/scipy/ndimage/src/nd_image.h#L67. I'll open a separate issue for that if it doesn't exist yet.

**Pull request merged**